From d89b308fb81b12ff2f452739d103cfee8043f94a Mon Sep 17 00:00:00 2001
From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>
Date: Fri, 14 Dec 2018 17:02:06 +0000
Subject: [PATCH 1/8] pc: 7.5 compat entries

RH-Author: Dr. David Alan Gilbert <dgilbert@redhat.com>
Message-id: <20181214170211.14267-2-dgilbert@redhat.com>
Patchwork-id: 83515
O-Subject: [RHEL8 qemu-kvm PATCH v3 1/6] pc: 7.5 compat entries
Bugzilla: 1655820
RH-Acked-by: Laurent Vivier <lvivier@redhat.com>
RH-Acked-by: Cornelia Huck <cohuck@redhat.com>
RH-Acked-by: Thomas Huth <thuth@redhat.com>

From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>

These are missing entries that are already in 7.6 from commits
64b860ac7db and 2f039646554 but are missing in the 3.1 rebase.

Signed-off-by: Dr. David Alan Gilbert <dgilbert@redhat.com>
Signed-off-by: Danilo C. L. de Paula <ddepaula@redhat.com>
---
 include/hw/i386/pc.h | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/include/hw/i386/pc.h b/include/hw/i386/pc.h
index f8f35af..efcb208 100644
--- a/include/hw/i386/pc.h
+++ b/include/hw/i386/pc.h
@@ -1008,6 +1008,22 @@ extern void igd_passthrough_isa_bridge_create(PCIBus *bus, uint16_t gpu_dev_id);
             .driver   = "Skylake-Server" "-" TYPE_X86_CPU,\
             .property = "clflushopt",\
             .value    = "off",\
+        },{ /* PC_RHEL7_5_COMPAT from PC_COMPAT_2_12 */ \
+            .driver   = TYPE_X86_CPU,\
+            .property = "legacy-cache",\
+            .value    = "on",\
+        },{ /* PC_RHEL7_5_COMPAT from PC_COMPAT_2_12 */ \
+            .driver   = TYPE_X86_CPU,\
+            .property = "topoext",\
+            .value    = "off",\
+        },{ /* PC_RHEL7_5_COMPAT from PC_COMPAT_2_12 */ \
+            .driver   = "EPYC-" TYPE_X86_CPU,\
+            .property = "xlevel",\
+            .value    = stringify(0x8000000a),\
+        },{ /* PC_RHEL7_5_COMPAT from PC_COMPAT_2_12 */ \
+            .driver   = "EPYC-IBPB-" TYPE_X86_CPU,\
+            .property = "xlevel",\
+            .value    = stringify(0x8000000a),\
         },
 
 
-- 
1.8.3.1

